### DISCUSSION TITLE
Instruction set table

### DISCUSSION THREAD

**Discussion by msinger** [emu-russia/dmgcpu#122] [General]
Please note that there are a few mistakes in the instruction set table that you have linked in sm83.md. (https://www.pastraiser.com/cpu/gameboy/gameboy_opcodes.html) I also used this table a lot. I like it, because it is so colorful. But you have to be aware that it is not fully true. Otherwise it can make you run in circles unnecessarily if the things you measure in reality do not fit. (Been there, done that. :disappointed: )

Also, some instructions have a different mnemonic than in the 1996 Sharp Databook.

1. `STOP` is actually only 1 byte, not 2. It should be always followed by a NOP instruction though, because of some bug.
2. The `LD (C), A` and `LD A, (C)` instructions at the bottom of column x2 actually are only 1 byte, not 2.
3. It is not true that the `SRA` instructions are setting `C` to 0. They are actually setting it to the LSB that falls out at the right side, just like described in the Sharp Manual. So they have the same flag behavior than the other shift/rotate instructions (`Z 0 0 C`).
4. The `BIT n, (HL)` instructions actually require only 12 cycles, not 16.

**Comment by ogamespec** [comment]
Thanks, I think it makes sense to make a "local affiliate" of this table. Or send all the comments to the author to fix it. After all, it looks beautiful :)
